File: Assignment13/Program2.c
#include<stdio.h>

void print_even_numbers(int limit)
{ 
    int iCnt = 0;
    iCnt = 2;
  while(iCnt <= limit )
  {
    if(iCnt % 2 == 0)
    {
    printf("%d\t",iCnt);
    }
    iCnt++;
  }
}
int main()
{
    int limit;

    printf("Enter number: ");
    scanf("%d",&limit);

    print_even_numbers(limit);
    return 0;
}

//Time Complexity = 0(N)

File: Assignment18/Program2.java
class Logic
{
    void printEvenNumbers(int n)
    {
        int iCnt = 0;

        for(iCnt = 2;iCnt <= n; iCnt++)
        {
          if(iCnt % 2 == 0)
        {
            System.out.println(iCnt);
        }
        }
        
    }
}
class Program2
{
    public static void main(String A[])
    {
        Logic obj = new Logic();
        obj.printEvenNumbers(20);
    }
}